pkg/types/savedview.go

```diff
@@ -9,12 +10,12 @@ type SavedView struct {
 
 	TimeAuditable
 	UserAuditable
-	OrgID      string `json:"orgId" bun:"org_id,notnull"`
-	UUID       string `json:"uuid" bun:"uuid,pk,type:text"`
-	Name       string `json:"name" bun:"name,type:text,notnull"`
-	Category   string `json:"category" bun:"category,type:text,notnull"`
-	SourcePage string `json:"sourcePage" bun:"source_page,type:text,notnull"`
-	Tags       string `json:"tags" bun:"tags,type:text"`
-	Data       string `json:"data" bun:"data,type:text,notnull"`
-	ExtraData  string `json:"extraData" bun:"extra_data,type:text"`
+	OrgID      string      `json:"orgId" bun:"org_id,notnull"`
+	UUID       valuer.UUID `json:"uuid" bun:"uuid,pk,type:text"`
```

Comment:
```suggestion
	UUID       valuer.UUID `json:"uuid" bun:"uuid,pk,type:text"`
```
Aren't we changing this name to ID? Or will that happen in the next PR?